python/GafferImageUI/ImageViewUI.py

```diff
@@ -1300,49 +1350,217 @@ def __toggle( self, *args ):
 
 		self.getPlug().setValue( not v )
 
-class _CompareCatalogueOutputPlugValueWidget( GafferUI.PlugValueWidget ) :
+class _CompareImageWidget( GafferUI.Frame ) :
 
-	def __init__( self, plug, **kw ) :
+	def __init__( self, plug ) :
 
-		self.__button = GafferUI.MenuButton(
-			image = "catalogueOutput1.png",
-			menu = GafferUI.Menu(
-				Gaffer.WeakMethod( self.__menuDefinition ),
-				title = "Catalogue Output",
-			)
+		GafferUI.Frame.__init__( self, borderWidth = 0, borderStyle = GafferUI.Frame.BorderStyle.None_ )
+
+		self._qtWidget().setFixedHeight( 15 )
+		self.__node = plug.node()
+		self.__scriptNode = plug.node()["in"].getInput().node().scriptNode()
+		self.__defaultNodeSet = Gaffer.StandardSet( [] )
+		self.__nodeSet = self.__defaultNodeSet
+		self.__catalogueOutput = 0
+
+		row = GafferUI.ListContainer( orientation = GafferUI.ListContainer.Orientation.Horizontal )
+		with row :
+
+			self.__bookmarkNumber = GafferUI.Label( horizontalAlignment=GafferUI.Label.HorizontalAlignment.Right )
+			self.__bookmarkNumber.buttonPressSignal().connect( Gaffer.WeakMethod( self.__showEditorFocusMenu ), scoped=False )
+
+			self.__icon = GafferUI.Button( hasFrame=False, highlightOnOver=False )
+			self.__icon._qtWidget().setFixedHeight( 13 )
+			self.__icon._qtWidget().setFixedWidth( 13 )
+			self.__icon.buttonPressSignal().connect( Gaffer.WeakMethod( self.__showEditorFocusMenu ), scoped=False )
+
+			# TODO - why does the pinning dropdown use collapsibleArrowDown instead of menuIndicator?
```

Comment:
I don't know - perhaps give it a try, and if it seems to work, change it?

Reply:
Added commit feef55b

Looks good to me, and Murray thought it was a good idea.

Reply:
We still need to remove the TODO from ImageViewUI.